polls/polls.js

```diff
@@ -0,0 +1,115 @@
+// import
+import { getPoll, createPoll, getUser, logout } from '../fetch-utils.js';
+
+import { renderPoll } from '../render.js';
+
+const pollForm = document.querySelector('.form');
+const plusOneButton = document.querySelector('#option-one-plus');
+const minusOneButton = document.querySelector('#option-one-minus');
+const plusTwoButton = document.querySelector('#option-two-plus');
+const minusTwoButton = document.querySelector('#option-two-minus');
+const endPollButton = document.querySelector('#end-poll-button');       
+const optionOneVotesEl = document.querySelector('#option-one-votes');
+const optionTwoVotesEl = document.querySelector('#option-two-votes');
+const optionOneTextEl = document.querySelector('#option-one-text');
+const optionTwoTextEl = document.querySelector('#option-two-text');
+const questionTextEl = document.querySelector('#question-text');
+// const currentPollEl = document.querySelector('#current-poll');
+const previousPollsEl = document.querySelector('#previous-polls');
+const logOutButton = document.getElementById('logout');
+
+//State
+
+let pollQuest = '';
+let optOneInput = '';
+let optTwoInput = '';
+let optOneVotes = 0;
+let optTwoVotes = 0;
+
+getUser();
+
+plusOneButton.addEventListener('click', () => {
+    optOneVotes++;
+
+    optionOneVotesEl.textContent = optOneVotes;
+});
+
+minusOneButton.addEventListener('click', () => {
+    optOneVotes--;
+    optionOneVotesEl.textContent = optOneVotes;
+});
+
+plusTwoButton.addEventListener('click', () => {
+    optTwoVotes++;
+    optionTwoVotesEl.textContent = optTwoVotes;
+});
+
+minusTwoButton.addEventListener('click', () => {
+    optTwoVotes--;
+    optionTwoVotesEl.textContent = optTwoVotes;
```

Comment:
for lines 34, 39, 44, and 49 we will want to use the `displayCurrentPoll()` function to let it do that work for you